### PR TITLE
Release v1.0.10

### PR DESCRIPTION
**Pull request by str4d**

### CODE REVIEW

**Comment by str4d** [comment]
@zkbot r+

**Comment by bitcartel** [comment]
ACK

File: doc/man/zcashd.1 [hunk 1]
```diff
@@ -72,9 +72,14 @@ Imports blocks from external blk000??.dat file on startup
...
 .HP
+\fB\-mempooltxinputlimit=\fR<n>
+.IP
+Set the maximum number of transparent inputs in a transaction that the
+mempool will accept (default: 0 = no limit applied)
+.HP
 \fB\-par=\fR<n>
 .IP
-Set the number of script verification threads (\fB\-4\fR to 16, 0 = auto, <0 =
+Set the number of script verification threads (\fB\-2\fR to 16, 0 = auto, <0 =
```
Comment (daira) [inline doc/man/zcashd.1]: Is this change just because the man page update script happened to be run on a machine/VM with two cores?

**Comment by str4d** [comment]
No, this was the pr that changed from virtual cores to real ones.

**Comment by daira** [comment]
Ah I see. The man page should still say something like  "-(number of cores)" rather than "-2" though.

**Pull request merged**